app/controllers/questions/publishQuestion.js

```diff
@@ -0,0 +1,35 @@
+import { db } from 'app/utils/db.server';
+import { PIN_QUESTION_ERROR_MESSAGE, INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE } from 'app/utils/constants';
+
+const publish = async (questionId) => {
```

Comment:
We could name the controller to something more clear like `setQuestionAsPublic` or `publishQuestion`

Reply:
Done ✅